app/controllers/application_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 class ApplicationController < ActionController::API
   include ActionController::HttpAuthentication::Token::ControllerMethods
+  include ActionView::Layouts
```

Comment:
Include this in the controller where it is used, because it probably won't be used elsewhere.

Reply:
Must be included in ActionController.